.github/workflows/testing.yml

```diff
@@ -91,4 +91,5 @@ jobs:
           examples/examples_test.sh
           security/advancedtls/examples/examples_test.sh
           interop/interop_test.sh
-          make testsubmodule
+          cd security/advancedtls && go test -cpu 1,4 -timeout 7m google.golang.org/grpc/security/advancedtls/...
+          cd ../../security/authorization && go test -cpu 1,4 -timeout 7m google.golang.org/grpc/security/authorization/...
```

Comment:
Do you know if there is a homedir or some sort of base dir that i can use to avoid the `../..`.
Or maybe I can do:
```
run: |
  export CURDIR=$(pwd)
  cd ${CURDIR}/security/advancedtls && go test -cpu 1,4 -timeout 7m google.golang.org/grpc/security/advancedtls/...
  cd ${CURDIR}/security/authorization && go test -cpu 1,4 -timeout 7m google.golang.org/grpc/security/authorization/...
```

Reply:
Or use subshells:

```
(cd <path> && go test ...)
(cd <path> && go test ...)
```

Or add a script that runs them.

Reply:
Ended up using the `GITHUB_WORKSPACE` which is provided by the [checkout action](https://github.com/actions/checkout)